### PR TITLE
feat: Add websocket handling

### PR DESCRIPTION
**Pull request by dignifiedquire**
Needs https://github.com/diasdavid/js-libp2p-swarm/pull/54

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,17 +21,39 @@ exports.Node = function Node (peerInfo) {
...
   this.swarm.connection.reuse()
 
-  this.start = (callback) => {
-    this.swarm.transport.listen('tcp', {}, null, callback)
-    // when we get websockets addr in our config
-    // this.swarm.transport.listen('tcp', {}, null, callback)
+  this.start = (done) => {
+    parallel(availableTransports.map((ts) => (cb) => {
+      // Listen on the given transport
+      this.swarm.transport.listen(ts, {}, null, cb)
```
Comment (diasdavid) [inline src/index.js]: can we actually have this filtering logic as a thing of `swarm`? 

`swarm.listen`. Other people will hit the same issue and it makes sense that swarm knows how to solve that.

Comment (dignifiedquire) [inline src/index.js]: Sure, how exactly do we want this to happen though? just immediately calling `cb` if the transport is not available? or calling back with an error?

Comment (diasdavid) [inline src/index.js]: Let's just call the cb(), since we want configs to be portable, but capabilities aren't

**Comment by diasdavid** [comment]
swarm 0.12.10 released

**Pull request merged**